samples/word/40-tables/alignment.yaml

```
@@ -0,0 +1,138 @@
+order: 3
+id: word-tables-alignment
+name: Get the alignment of a table
+description: 'Get alignment details of a table, a table row, and a table cell.'
+author: OfficeDev
+host: WORD
+api_set:
+    WordApi: '1.3'
+script:
+    content: |
+        $("#get-table-alignment").click(() => tryCatch(getTableAlignment));
+        $("#get-table-row-alignment").click(() => tryCatch(getTableRowAlignment));
+        $("#get-table-cell-alignment").click(() => tryCatch(getTableCellAlignment));
+        $("#setup").click(() => tryCatch(insertTable));
+
+        async function getTableAlignment() {
+          // Get alignment details about the first table in the document.
+          await Word.run(async (context) => {
+            const firstTable = context.document.body.tables.getFirst();
+            firstTable.load(["horizontalAlignment", "verticalAlignment"]);
+            await context.sync();
+
+            console.log(`Details about the alignment of the first table:`);
+            console.log(`Horizontal: ${firstTable.horizontalAlignment}`);
+            console.log(`Vertical: ${firstTable.verticalAlignment}`);
+          });
+        }
+
+        async function getTableRowAlignment() {
+          // Get alignment details about the first row of the first table in the document.
+          await Word.run(async (context) => {
+            const firstTableRow = context.document.body.tables.getFirst().rows.getFirst();
```

Consider splitting this out a bit so it's more readable.
```suggestion
            const firstTable = context.document.body.tables.getFirst();
            const firstTableRow = firstTable.rows.getFirst();
```